hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/connector/HazelcastWriters.java

```diff
@@ -152,6 +158,57 @@ public static ProcessorMetaSupplier writeListSupplier(@Nonnull String name, @Nul
         ));
     }
 
+    @Nonnull
+    public static ProcessorMetaSupplier writeReliableTopicSupplier(@Nonnull String name,
+                                                                   @Nullable ClientConfig clientConfig) {
+        boolean isLocal = clientConfig == null;
+        return preferLocalParallelismOne(new WriterSupplier<>(
+                asXmlString(clientConfig),
+                ArrayList::new,
+                ArrayList::add,
+                instance -> {
+                    ITopic<Object> topic = instance.getReliableTopic(name);
+                    return buffer -> {
+                        try {
+                            for (Object item : buffer) {
+                                topic.publish(item);
```

Comment:
We don't need the buffer when we add items one by one. Btw, couldn't we use the same `addAllAsync` here?

Reply:
Topic doesn't have a `publishAll` type of method (as of now). We shouldn't use the underlying `Ringbuffer` directly because that code will become obsolete silently if the topic implementation changes. So I think we are stuch with one-by-one publishing. 

In that case we indeed don't need the buffer, but is it worth removing? That would mean not using `WriteBufferedP`, so would need to write a new processor for not much of a benefit. Or am I missing something?

Reply:
You can use:
```java
return preferLocalParallelismOne(() -> new AbstractProcessor() {
    protected boolean tryProcess(int ordinal, @Nonnull Object item) {
        topic.publish(item);
    }
});
```

Reply:
Or better: 
```java
        new SinkBuilder<ITopic<T>, T>("reliableTopic(" + name + "))", 
                        ctx -> ctx.jetInstance().getReliableTopic(name))
                .receiveFn((ctx, item) -> ctx.publish(item))
                .build();
```

Reply:
Nice! Did so.